cli-helpers/src/commands/invoke-avl-datafeed-validator.ts

```diff
@@ -1,11 +1,13 @@
 import { Command } from "@commander-js/extra-typings";
 import inquirer from "inquirer";
-import { STAGES, STAGE_OPTION, invokeLambda } from "../utils";
+import { STAGES, STAGE_OPTION_WITH_DEFAULT, getSecretByKey, invokeLambda } from "../utils";
 
 export const invokeAvlDataFeedValidator = new Command("invoke-avl-data-feed-validator")
-    .addOption(STAGE_OPTION)
+    .addOption(STAGE_OPTION_WITH_DEFAULT)
```

Comment:
I think this is better off as `STAGE_OPTION` as Deanna wanted some of the AVL cli helpers to explicitly ask for the stage rather than fall back to the local env